client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -261,8 +261,8 @@ const VaccineCourseDoseTable = ({
           __typename: 'VaccineCourseDoseNode',
           id: FnUtils.generateUUID(),
           label: `${courseName} ${doses.length + 1}`,
-          minAgeMonths: (previousDose?.minAgeMonths ?? 0) + 1,
-          maxAgeMonths: (previousDose?.minAgeMonths ?? 0) + 2,
+          minAgeMonths: previousDose?.maxAgeMonths ?? 0,
+          maxAgeMonths: (previousDose?.maxAgeMonths ?? 0) + 1,
```

Comment:
I wonder whether this would be more a helpful default... so if the first range was 0-6mnths, next would be 6-12, rather than 6-7?
```ts
  const addDose = () => {
    const previousDose = doses[doses.length - 1];

    const previousMin = previousDose?.minAgeMonths ?? 0;
    const previousMax = previousDose?.maxAgeMonths ?? 0;
    const previousRange = previousMax - previousMin;

    updatePatch({
      vaccineCourseDoses: [
        ...doses,
        {
          __typename: 'VaccineCourseDoseNode',
          id: FnUtils.generateUUID(),
          label: `${courseName} ${doses.length + 1}`,
          minAgeMonths: previousMax,
          maxAgeMonths: previousMax + (previousRange || 1),
          minIntervalDays: previousDose?.minIntervalDays ?? 30,
        },
      ],
    });
  };
```

Reply:
Yeah, sure. Hard to know for sure without knowing real-world cases, but this seems reasonable: 8cdd1b7520